pw-cli: add page (#8029)
tldr-pages/tldr: pages/linux/pw-cli.md

# pw-cli

> The PipeWire Command Line Interface.

> > More information: <https://docs.pipewire.org/page_man_pw_cli_1.html>.

Reviewer: @navarroaxel, @CleanMachine1, what about using [manned.org][doc]? The provided link referrers to a page with not user-friendly formatted page

[doc]: https://manned.org/pw-cli.1#head6

> - List all nodes (sinks and sources) along with their IDs:

Reviewer: ```suggestion
- Print all nodes (sinks and sources) along with their IDs:
```

> `pw-cli list-objects Node`

Reviewer: What is `Node` here? Is it a placeholder or not? I don't see any info about `Node` [here][manned]. I expected smth like `Node` or `list-objects [arguments...]`.

[manned]: https://manned.org/pw-cli.1#head6

Author: It's a keyword. A type. Possible values are `pw-cli list-objects Node` and `pw-cli list-objects Device`.

Reviewer: What about this change:

```suggestion
`pw-cli list-objects {{Node|Device}}`
```

? You can also modify your description as: `List all nodes/devices (sinks and sources) along with their IDs:`.

Author: It'd be `nodes (sinks and sources)/devices`, sources and sinks are types of node. 

After taking a better look at it, possible values are anything from https://docs.pipewire.org/page_objects_design.html. Considering it's too much to list, I'd stick with the original text. Users most likely want to operate on nodes.

> - Print information about an object with the specified ID:

Reviewer: ```suggestion
- Print information about an object with a specific ID:
```

`pw-cli info {{4}}`

> - Print all objects' information:

Reviewer 2: ```suggestion
- Print all object information:
```

Author: It doesn't print all information about single object but information about every object. All nodes and devices.
